lib/matplotlib/tests/test_font_manager.py

```diff
@@ -215,3 +215,20 @@ def test_missing_family(caplog):
         "findfont: Generic family 'sans' not found because none of the "
         "following families were found: this-font-does-not-exist",
     ]
+
+
+def test_fontcache_thread_safe():
```

Comment:
I don't think this test is isolated enough; if threading is broken it might cause breakage in random other tests. This probably needs a subprocess to be less error-prone.

Reply:
The failure mode is "segfault" so definitely not isolated enough, will push it to a sub process tomorrow.

Reply:
tests are now run in a sub-process so as to not leak into or seg-fault the test suite.